dev-python/randomize/randomize-0.14.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{4,7,8} )
```

Comment:
python3_4 is not used anymore. New packages must be tested at least against 7, 8 and 9. Same thing for other ebuilds